packages/strapi-icons/vite.config.ts

```diff
@@ -22,5 +23,5 @@ export default defineConfig({
       },
     },
   },
-  plugins: [react()],
+  plugins: process.env.DTS !== 'true' ? [react()] : [dts(), react()],
```

Comment:
see other comment, we can just use the plugin, there's no need for a flag because we were already publishing types.

Reply:
ok